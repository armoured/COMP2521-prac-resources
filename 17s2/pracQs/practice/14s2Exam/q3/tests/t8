./q3 < tests/t8.in
