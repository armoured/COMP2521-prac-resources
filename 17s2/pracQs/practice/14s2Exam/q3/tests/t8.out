Graph:
# vertices = 6, # edges = 5

0 is connected to 1 2 3 4 5
1 is connected to 0
2 is connected to 0
3 is connected to 0
4 is connected to 0
5 is connected to 0

Well-connected vertices
0 has 5 connections
